docs/data/material/customization/typography/typography.md

```diff
@@ -241,7 +244,49 @@ const theme = createTheme({
 });
 ```
 
-**Step 2. Update the necessary typings (if you are using TypeScript)**
+**Step 2. (Optional) Set the default semantic element for your new variant**
+
+At this point, you can already use the new `poster` variant, which will render a `<span>` by default with your custom styles. Sometimes you may want to default to a different HTML element for semantic purposes, or to replace the inline `<span>` with a block-level element for styling purposes.
+
+To do this we need to update the `variantMapping` prop of the `Typography` globally in the theme:
```

Comment:
Try to avoid "we," and instead phrase each step in terms of the action that the reader/user is taking.

```suggestion
To do this, update the `variantMapping` prop of the Typography component globally, at the theme level:
```